### DISCUSSION TITLE
Hash mode

### DISCUSSION THREAD

**Discussion by Slepwin** [loxilb-io/loxilb#221] [Q&A]
Could you please describe hash mode in details. What algorithms is used to get hash? For example 2 loxilb without HA sync (just anycast vip announce) getting the same hash for same 5-tuple and balance to the same endpoints?

**Comment by UltraInstinct14** [comment]
Hi @Slepwin 

We reuse a hash value provided by Linux kernel [hash function](https://elixir.bootlin.com/linux/v5.15/source/net/core/flow_dissector.c#L1514) . Although our team/community has not verified its consistency in a HA scenario, we will soon confirm this.

Thanks !!

**Reply by UltraInstinct14** [reply]
@Slepwin  We have verified consistent hashing in a clustered/HA LB environment with L3-DSR mode.  It required a little fix to our code. A new release will be made available soon.
